pages/docs/relayer/running-relayer/quick-start.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: Quick Start
+description: Get up to speed for running the relayer.
+---
+
+import Callout from "../../../../components/Callout";
+import { Tabs, Tab } from "../../../../components/Tabs";
+
+# Quick Start
+
+You can spin up the Relayer in a few simple commands. This guide will walk you through the steps to get started.
+
+By the end of this guide, you will have a running relayer that is publicly accessible via HTTPS endpoint behind a reverse proxy.
+
+## Prerequisites
+
+- A Unix-based operating system (Linux, macOS, WSL, etc.)
+- curl
+
+## Installation
+
+In your terminal, run the following command to download the latest version of the relayer:
+
+```bash
+curl -fsSL https://git.io/get-webb-relayer.sh | sh
+```
+
+To install a specific version of the relayer, run the following command:
+
+```bash
+curl -fsSL https://git.io/get-webb-relayer.sh | sh -s <version>
+```
+
+This will download the relayer binary (or update it if it already exists) and place it in your `$HOME/.webb` directory. The script
+will also suggest adding the directory to your `PATH` environment variable.
+
+### Verify Installation
+
+To verify that the relayer was installed successfully, run the following command:
+
+```bash
+~/.webb/webb-relayer --version
+```
+
+## Configuration
+
+Before running the relayer, you will need to configure it. The relayer could be configured by using TOML or JSON files along with environment variables.
+The Relayer will look for configuration files in the default locations, but you can also specify the path to the configuration file using the `--config` flag.
+
+- Default locations for configuration files:
+
+  - `$HOME/.config/webb-relayer/` (Linux)
+  - `$HOME/Library/Application Support/tools.webb.webb-relayer/` (macOS)
+
+For simplicity, we will use the TOML configuration file format in this guide and we will place the configuration file in the default locations.
+
+### Download the Example Configuration File
+
+To download the example configuration file, run the following command:
+
+<Tabs items={["Linux/WSL", "macOS"]} storageKey="selected-binary">
+  <Tab>
+
+    ```bash
+    curl -fsSL https://raw.githubusercontent.com/webb-tools/webb-relayer/master/config/example/config.toml -o ~/.config/webb-relayer/config.toml
+    ```
+
+  </Tab>
+  <Tab>
+
+    ```bash
+    curl -fsSL https://raw.githubusercontent.com/webb-tools/webb-relayer/master/config/example/config.toml -o ~/Library/Application\ Support/tools.webb.webb-relayer/config.toml
+    ```
+
+  </Tab>
+</Tabs>
+
+### Configure the Relayer
+
+Open the configuration file in your favorite text editor and take a look at the configuration options. The configuration file is well-documented and should be self-explanatory.
```

Comment:
We should include a screenshot here to further emphasize what the fetched config should appear as.